.github/actions/synthetics-stack-e2e/action.yml

```diff
@@ -0,0 +1,159 @@
+---
+name: synthetics-stack-e2e
+description: |
+  An action that orchestrates the Synthetics Stack E2E tests for a Kibana PR.
+  It first creates a Kibana docker image, then it deploys an ESS cluster with the created docker image,
+  and finally it runs the Synthetics Stack E2E tests against the deployed ESS cluster.
+
+inputs:
+  buildkite-pipeline-slug:
+    description: 'The Buildkite pipeline slug'
+    default: kibana-pr-synthetics-stack-e2e-ci
+    required: false
+  vault-url:
+    description: 'Vault URL'
+    required: true
+  vault-role-id:
+    description: 'Vault role ID'
+    required: true
+  vault-secret-id:
+    description: 'Vault secret ID'
+    required: true
+  comment-id:
+    description: 'The GitHub Comment ID'
+    default: ${{ github.event.comment.id }}
+  commit-sha:
+    description: 'The commit SHA'
+    default: ${{ github.event.pull_request.head.sha }}
+    required: false
+  pr-number:
+    description: 'The PR number'
+    default: ${{ github.event.issue.number }}
+    required: false
+  user:
+    description: 'The GitHub user that triggered the workflow'
+    default: ${{ github.triggering_actor }}
+    required: true
+  context:
+    description: 'The commit status context'
+    default: synthetics-stack-e2e
+    required: false
+  comment-command:
+    description: 'The comment command'
+    default: '/synthetics-stack-e2e'
+    required: false
+
+runs:
+  using: composite
+  steps:
+
+    - uses: elastic/apm-pipeline-library/.github/actions/github-token@current
+      with:
+        url: ${{ inputs.vault-url }}
+        roleId: ${{ inputs.vault-role-id }}
+        secretId: ${{ inputs.vault-secret-id }}
+
+    - name: Is an Elastic member?
+      id: is_elastic_member
+      uses: elastic/apm-pipeline-library/.github/actions/is-member-elastic-org@current
+      with:
+        username: ${{ inputs.user }}
+        token: ${{ env.GITHUB_TOKEN }}
+
+    - name: Notify with a reaction
+      continue-on-error: true
+      uses: elastic/apm-pipeline-library/.github/actions/comment-reaction@current
```

Comment:
what if a comment is added by a non-elastician user?

Reply:
`emoji: ${{ steps.is_elastic_member.outputs.result && '+1' || '-1' }}`

It will be a thumbs down.